adapters/adapterstest/test_json.go

```diff
@@ -18,6 +20,8 @@ import (
 	"github.com/yudai/gojsondiff/formatter"
 )
 
+const JsonFileExtension string = ".json"
```

Comment:
`JsonFileExtension` can be private 

```
const jsonFileExtension = ".json"
```

Reply:
Good catch. No need to export those. Modified